code/build/package.json

```diff
@@ -25,7 +25,7 @@
     "@types/gulp-sourcemaps": "^0.0.32",
     "@types/jws": "^3.2.10",
     "@types/mime": "0.0.29",
-    "@types/minimatch": "^3.0.3",
+    "@types/minimatch": "^3.0.5",
```

Comment:
AFAIU `@types/minimatch` contains type declarations, not executable code, more important to update `minimatch` package

Reply:
I checked that actually `3.1.2` version is used in the lock file - so - NP with https://github.com/che-incubator/che-code/blob/main/code/package.json#L195.